### PR TITLE
osemgrep: align "files skipped" output with pysemgrep

### PR DESCRIPTION
**Pull request by hannesm**
PR checklist:

- [x] Purpose of the code is [evident to future readers](https://semgrep.dev/docs/contributing/contributing-code/#explaining-code)
- [x] Tests included or PR comment includes a reproducible test plan
- [x] Documentation is up-to-date
- [x] A changelog entry was [added to changelog.d](https://semgrep.dev/docs/contributing/contributing-code/#adding-a-changelog-entry) for any user-facing change
- [x] Change has no security implications (otherwise, ping security team)

If you're unsure about any of this, please see:

- [Contribution guidelines](https://semgrep.dev/docs/contributing/contributing-code)!
- [One of the more specific guides located here](https://semgrep.dev/docs/contributing/contributing/)

### CODE REVIEW

File: src/osemgrep/cli_scan/Scan_subcommand.ml [hunk 1]
```diff
@@ -240,55 +240,70 @@ let pp_skipped ppf
...
   Fmt.pf ppf "@.";
 
-  Fmt.pf ppf
-    " Skipped by limiting to files smaller than \
-     {self.target_manager.max_target_bytes} bytes:@.";
-  Fmt.pf ppf " (Adjust with the --max-target-bytes flag)@.@.";
+  Fmt.pf ppf " %a@. %a@.@."
+    Fmt.(styled `Bold string)
+    "Skipped by limiting to files smaller than \
+     {self.target_manager.max_target_bytes} bytes:"
```
Comment (aryx) [inline src/osemgrep/cli_scan/Scan_subcommand.ml]: this needs to be the actual value of the conf, not this self.xxx useless string.

**Review by aryx — APPROVED** [review]
Do we get some tests passing thanks to this?
Test plan?

File: src/osemgrep/cli_scan/Scan_subcommand.ml (hunk 1 above)
Comment (hannesm) [reply src/osemgrep/cli_scan/Scan_subcommand.ml]: done in 6fa4f920c

**Comment by hannesm** [comment]
> Do we get some tests passing thanks to this? Test plan?

we do not yet get passing tests from this, but getting closer to the pysemgrep output.

**Pull request merged**